br/README.md

```diff
@@ -30,7 +30,8 @@
 To build binary and run test:
 
 ```bash
-$ make
+$cd ../tidb
```

Comment:
Or it looks like access the variable `$cd`.
```suggestion
$ cd ../tidb
```

Reply:
(Also, it seems there isn't any example output in this case. Perhaps just removing the dollar will be better?)

Reply:
maybe, but I see no difference here. As long as no ambiguity will be induse here, I prefer just leave it be.